src/app/components/faq/FaqQuestion/faqQuestion.js

```diff
@@ -0,0 +1,21 @@
+
+
+import { useState } from 'react'
+
+export default function FaqQuestion(props) {
```

Comment:
Change the file to .tsx and assign a type to the props to make this more type safe, you can see some of the other components for examples

Reply:
done